src/types/board-vulnerability.ts

```diff
@@ -0,0 +1,20 @@
+import { Vulnerability } from './vulnerability';
+/** Board Vulnerability from NS perspective */
+export const BOARD_TO_VULNERABILITY: Record<number, Vulnerability> = {
+	1: Vulnerability.NvNv,
+	2: Vulnerability.VNv,
+	3: Vulnerability.NvV,
+	4: Vulnerability.VV,
+	5: Vulnerability.VNv,
+	6: Vulnerability.NvV,
+	7: Vulnerability.VV,
+	8: Vulnerability.NvNv,
+	9: Vulnerability.NvV,
+	10: Vulnerability.VV,
+	11: Vulnerability.NvNv,
+	12: Vulnerability.VNv,
+	13: Vulnerability.VV,
+	14: Vulnerability.NvNv,
+	15: Vulnerability.VNv,
+	16: Vulnerability.NvV,
```

Comment:
This probably needs to be case 0, which I assume is what the `if` tries to solve in another file.